src/Ada.ts

```diff
@@ -120,11 +120,11 @@ async function interact<T>(
  */
 export class Ada {
   /** @ignore */
-  transport: Transport;
+  transport: Transport<string>;
```

Comment:
The `Transport` takes a generic argument that defaults to `string`.

In Typscript, if you don't specify a generic parameter, it just uses the default. However, Flow doesn't have this feature. Easiest solution is to just specify the default argument.